upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -98,6 +100,9 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap) {
 	dest["EncodeGCELabel"] = gce.EncodeGCELabel
 
 	dest["DnsControllerImage"] = tf.DnsControllerImage
+
+	// TODO: Only for AWS?
+	dest["AWSRegion"] = tf.AWSRegion
```

Comment:
Looks like it is trivial! In fact, there's already a Template Function called "Region". I'm in luck! Thanks for the catch :)

https://github.com/kubernetes/kops/blob/2c93bff6d73b1f3e7e0d0a422a7710cc3777b3b9/upup/pkg/fi/cloudup/apply_cluster.go#L502